src/components/Fetch/search.spec.js

```diff
@@ -83,4 +80,19 @@ test('Fetch-Search', function() {
     <Button>Prev</Button>
   ]), false);
   assert.equal(wrapper.find('.not-found').exists(), true);
+
+  wrapper.find('#findInput').instance().value = 'ASIO';
+  wrapper.find('#findInput').at(0).simulate('change', {});
+  wrapper.instance().state.caseSensitive = true;
+
+  assert.equal(wrapper.state('findIdx'), 0);
+  assert.equal(wrapper.state('findResults').length, 10);
+  assert.equal(wrapper.state('find'), 'ASIO');
+  assert.equal(wrapper.state('wrap'), false);
+  assert.equal(wrapper.state('caseSensitive'), true);
+  assert.equal(wrapper.state('detailsOpen'), false);
+  assert.equal(wrapper.containsAllMatchingElements([
```

Comment:
I'd recommend writing this expression as assert.ok(wrapper.contains....). This is not a universally true statement. For example, I wouldn't write the asserts testing state booleans with assert.ok()